.jenkinsci/linux-post-step.groovy

```diff
@@ -0,0 +1,24 @@
+def linuxPostStep() {
+  timeout(time: 600, unit: "SECONDS") {
+    try {
+      if (currentBuild.result != "UNSTABLE") {
+        if (BRANCH_NAME ==~ /(master|develop)/) {
```

Comment:
can't it be like `if (currentBuild.result != "UNSTABLE" && BRANCH_NAME ==~ /(master|develop)/)`?